tests/unittest/_torch/sampler/test_return_top_k_logprobs.py

```
@@ -0,0 +1,96 @@
+def test_generate_with_top_logprobs_and_disabler_logprobs(
+        llm_torch, input_prompts):
+    max_tokens = 8
+    top_logprobs = 2
+    top_k = 2
+    top_p = None
+
+    # expected to fail testcases
+    with pytest.raises(
+            ValueError,
+            match=".*You need to set logprobs to True to use top_logprobs.*"):
+        sampling_params = SamplingParams(max_tokens=max_tokens,
```

Ruff (0.12.2)

Local variable `sampling_params` is assigned to but never used

Remove assignment to unused variable `sampling_params`

(F841)